modules/con_schema.adoc

```diff
@@ -256,6 +256,10 @@ All fields are optional unless otherwise marked.
 * **SESSION_COOKIE_SECURE** [boolean]: Whether the `secure` property should be set on session cookies. Defaults to False. Recommended to be True for all installations using SSL.
 ** **Example**: True
 ** **Reference**: https://en.wikipedia.org/wiki/Secure_cookies
+* **SSL_PROTOCOLS** [array] required: If specified, nginx is configured to enabled a list
```

Comment:
Does `required` mean the field is required? I don't think it is.

Reply:
I got the information from here: https://issues.redhat.com/browse/PROJQUAY-691 I agree, though. Starting with "If specified" doesn't sound like it's required. I'll just take off the "required" if that seems okay to you.